README.md

```diff
@@ -1,3 +1,57 @@
-# taoPublishing
+# TAO _taoPublishing_ extension
 
-Pushes a test to a delivery environment
+![TAO Logo](https://github.com/oat-sa/taohub-developer-guide/raw/master/resources/tao-logo.png)
+
+![GitHub](https://img.shields.io/github/license/oat-sa/extension-tao-publishing.svg)
+![GitHub release](https://img.shields.io/github/release/oat-sa/extension-tao-publishing.svg)
+![GitHub commit activity](https://img.shields.io/github/commit-activity/y/oat-sa/extension-tao-publishing.svg)
+
+> Pushes a test to a delivery environment
+
+
+## Installation instructions
+
+These instructions assume that you have already a TAO installation on your system. If you don't, go to
+[package/tao](https://github.com/oat-sa/package-tao) and follow the installation instructions.
+
+
+Add the extension to your TAO composer and to the autoloader:
+```bash
+$ composer require oat-sa/extension-tao-publishing ^2.1 --prefer-source
+```
+
+Install the extension on the CLI from the project root:
+
+**Linux:**
+```bash
+sudo php tao/scripts/installExtension.php taoPublishing
+```
+
+**Windows:**
+```bash
+php tao\scripts\installExtension.php taoPublishing
+```
+
+**Docker**
+```bash
+$ docker container exec -it {YOUR-TAO-INSTANCE-PHP-CONTAINER} php tao/scripts/installExtension.php taoPublishing
+```
+
+After the install just log out and log in again on the TAO Platform.
+
+As a system administrator you can also install it through the TAO Extension Manager:
+- Settings (the gears on the right hand side of the menu) -> Extension manager
+- Select _taoPublishing_ on the right hand side, check the box and hit _install_
+
+##Set up deliveries on TAO Platform
```

Comment:
I think you need and extra blank line because it's showing `##`

Reply:
Fixed.